### PR TITLE
small typo fix in documentation

### PR DESCRIPTION
**Pull request by novadeviator**
a missing ```::method``` and additional ```code::``` blocks was including next method explanation as part of code instead of breaking out

## Purpose and Motivation
just fixing some typos in documentation
## Types of changes

- Documentation

## To-do list

- [ ] Code is tested
- [ ] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

File: HelpSource/Classes/Array.schelp [hunk 1]
```diff
@@ -365,10 +365,11 @@ Routine {
 	a.fork; \doneAll.postln;
 }.play;
 )
+::
 
-poll(trig, label, trigid)
+method::poll(trig, label, trigid)
```
Comment (telephon) [inline HelpSource/Classes/Array.schelp:370]: no need to add the argument names explicitly, and better not. The help system does this for us.

File: HelpSource/Classes/Array.schelp [hunk 2]
```diff
@@ -365,10 +365,11 @@ Routine {
...
 apply an array of Poll units to an array of UGens (see those helpfiles for more details).
-
+code::
```
Comment (telephon) [inline HelpSource/Classes/Array.schelp:372]: you can also remove the `s.boot;`. A booted server is assumed to be the case anyway.

**Review by telephon — CHANGES_REQUESTED** [review]
Thank you!

File: HelpSource/Classes/Array.schelp (hunk 1 above)
Comment (novadeviator) [reply HelpSource/Classes/Array.schelp]: ok. done.

File: HelpSource/Classes/Array.schelp (hunk 2 above)
Comment (novadeviator) [reply HelpSource/Classes/Array.schelp:372]: done.

**Comment by brianlheim** [comment]
@novadeviator could you please rebase this branch on develop? we are no longer accepting PRs to 3.11. see instructions here: https://github.com/supercollider/supercollider/wiki/Creating-pull-requests#rebasing-to-change-the-base-branch-of-a-pr-branch

**Comment by novadeviator** [comment]
have I done this correctly?
rebasing scares the **** out of me, sorry. I have no clue if I'm doing this right.

**Comment by brianlheim** [comment]
no, this is not correct. if you tell me what commands you ran, I can help you. or, you can submit a new PR based off develop which contains only the commits you authored.

**Comment by novadeviator** [comment]
I created a new PR at https://github.com/supercollider/supercollider/pull/5354 
I hope I have prepared it better. I followed Eli's guide in docs on wiki.

**Comment by brianlheim** [comment]
thanks!

**Pull request closed without merging**